packages/rendering/src/getPreviewWebpackConfig.ts

```diff
@@ -39,7 +39,7 @@ export function getPreviewWebpackConfig(examFilename: string, options: Rendering
       quiet: true,
       contentBase: [path.resolve(__dirname, '../public'), path.dirname(examFilename)],
       historyApiFallback: true,
-      port: 0,
+      port: options.usePort ? options.usePort : 0,
```

Comment:
This can be shortened a bit.

```suggestion
      port: options.usePort ?? 0,
```

Reply:
Done.